src/sagemaker/tuner.py

```diff
@@ -489,6 +604,82 @@ def _validate_parameter_ranges(self):
                     except KeyError:
                         pass
 
+    def transfer_learning_tuner(self, additional_parents=None, estimator=None):
+        """Creates a new ``HyperarameterTuner`` by copying the request fields from the provided parent to the new
```

Comment:
many instances of this typo in different places in this PR. should be "HyperparameterTuner"

Reply:
done